stablehlo/reference/InterpreterOps.cpp

```diff
@@ -123,6 +123,10 @@ SmallVector<InterpreterValue> evalRunParallelOp(
 
   SmallVector<InterpreterValue> results;
   for (auto &future : futures) results.append(future.get());
+
+  while (!processGrid.outfeedEmpty())
+    for (auto outfeed : processGrid.popOutfeed()) results.push_back(outfeed);
```

Comment:
I don't think this design is going to work because it is not going to always be possible to statically determine the number of entries expected in the outfeed (that looks equivalent to solving the halting problem).

Reply:
Let's open a ticket for this, leave a TODO in code and move on I think. The changes in this PR are already valuable on their own, and we can iterate on testing outfeeds in follow-up PRs.

Reply:
Opened #1725